libs/external-db-mysql/src/mysql_schema_provider.ts

```diff
@@ -78,4 +85,51 @@ export default class SchemaProvider implements ISchemaProvider {
         row.type = this.sqlSchemaTranslator.translateType(row.type)
         return row
     }
+
+    columnCapabilitiesFor(columnType: string): ColumnCapabilities {
+        switch (columnType) {
+            case 'text':
+            case 'url':
+                return {
+                   sortable: true,
+                   columnQueryOperators: [eq, ne, string_contains, string_begins, string_ends, include, gt, gte, lt, lte]
+                }
+            case 'number':
+                return {
+                    sortable: true,
+                    columnQueryOperators: [eq, ne, gt, gte, lt, lte, include]
+                }
+            case 'boolean':
+                return {
+                    sortable: true,
+                    columnQueryOperators: [eq]
+                }
+            case 'image':
+                return {
+                    sortable: true,
+                    columnQueryOperators: []
+                }
+            case 'object':
+                return {
+                    sortable: true,
+                    columnQueryOperators: [eq, ne]
+                }
+            case 'datetime':
+                return {
+                    sortable: true,
+                    columnQueryOperators: [eq, ne, gt, gte, lt, lte]
+                }
+
+            default:
+                throw new Error(`${columnType} - Unsupported field type`)
```

Comment:
I think all that switch-case should be in another file somewhere (even utils, but I'm not sure) and should be tested.

Reply:
maybe it should be in the mysql_capabilities file - a file that will be found in every db implementation that lists the capabilities of the db in one file.